src/Plot/Scale.elm

```diff
@@ -0,0 +1,32 @@
+module Plot.Scale where
+
+import Private.Scale.Linear as LinearScale
+import Private.Scale.OrdinalBands as OrdinalBands
+import Private.Extras.Set as Set exposing (Set)
+import Private.Scale exposing (Scale)
+
+type alias LinearScale = Scale Set Float
```

Comment:
I would rename the import then just to ensure no confusion between the two occurs.